src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$main-accent: #060B35;
+$secondary-color: #616070;
+$blue-accent: #00ACDC;
+$border-color: #F3F3F3;
+$background-color: #FFF;
+$link-text-color: #FFF;
```

Comment:
Why did you create two variables with the same values?
Also white color is default color
```suggestion
$background-color: #FFF;
```

Reply:
When I was copying variables from another task, I forgot to remove the duplicates. Will be fixed